src/howto.rst

```diff
@@ -557,3 +557,4 @@ To initiate the multisimulation,
 
        $ mpiexec -n #ofreplicas GOMC_xxx_yyyy <optional#ofthreads> conffile 
 
+.. Note:: As of right now, restarting is not supported for multisims. 
```

Comment:
Move this note to How to Restart section.

Reply:
![image](https://user-images.githubusercontent.com/39970712/72402751-22c36a00-371e-11ea-83d0-bd77b0c1e972.png)

Reply:
![image](https://user-images.githubusercontent.com/39970712/72402785-3bcc1b00-371e-11ea-8293-ff14edd54a51.png)